src/templates/Navigator/index.js

```diff
@@ -123,9 +123,12 @@ function Navigator(props) {
                 </Sticky>
               </div>
             </Col>
-            <div className={styles['contact-card']}>
-              <ContactCard {...contactProps} />
-            </div>
+            {
+              contactProps &&
```

Comment:
Is there ever a chance that this will not be a falsey value, but still invalid props?

Reply:
well it could be but there wasn't a check for invalid props before this change - i don't think the component would render if they were invalid

the point of this check was to determine if to show it at all